### PR TITLE
alcove 1.0.3 (new cask)

### PR DESCRIPTION
**Pull request by chirpcel**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/a/alcove.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  url "https://releases.tryalcove.com/latest/Alcove.dmg"
+  name "Alcove"
+  desc "Utility to add Dynamic Island like features to notch area"
+  homepage "https://tryalcove.com/"
+
+  livecheck do
+    url "https://github.com/henrikruscon/alcove-releases.git"
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```
Comment (khipp) [inline Casks/a/alcove.rb:13]:
```suggestion
```

We can omit the `livecheck` block and use the default `Git` strategy, which matches the Git tags.

File: Casks/a/alcove.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+cask "alcove" do
+  version "1.0.3"
+  sha256 :no_check
+
+  url "https://releases.tryalcove.com/latest/Alcove.dmg"
```
Comment (khipp) [inline Casks/a/alcove.rb:5]:
```suggestion
  url "https://github.com/henrikruscon/alcove-releases/releases/download/#{version}/Alcove.zip",
      verified: "github.com/henrikruscon/alcove-releases/"
```

File: Casks/a/alcove.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+cask "alcove" do
+  version "1.0.3"
+  sha256 :no_check
```
Comment (khipp) [inline Casks/a/alcove.rb:3]:
```suggestion
  sha256 "3f1c1f3400421d0ebbffa0aede126e2476d9ab72d12b06cbcebbe7c6c59a5fca"
```

**Review by khipp — COMMENTED** [review]
Thank you for your first contribution to `homebrew-cask`, @chirpcel! 🎉

I left a few suggestions below.

**Comment by chirpcel** [comment]
Thanks for your review @khipp ! I applied the suggestions, but now it says that the GitHub repository is not notable enough. Anyway to circumvent that? The repo is intended for the auto updater by the application, so I think it won't change any time.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**